partcad/src/partcad/wrappers/wrapper_obj.py

```
@@ -0,0 +1,66 @@
+from OCP.gp import gp_Pnt
+
+sys.path.append(os.path.dirname(__file__))
+from . import wrapper_common
+
+def process(path, request):
+    try:
+        vertices = []
+        faces = []
+
+        # Read the OBJ file
+        with open(path, 'r') as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)